mappings/net/minecraft/entity/living/LivingEntity.mapping

```diff
@@ -7,15 +8,21 @@ CLASS net/minecraft/class_154 net/minecraft/entity/living/LivingEntity
 	FIELD field_557 hurtTime I
 	FIELD field_560 deathTime I
 	FIELD field_561 attackTime I
+	FIELD field_567 outofCombatTime I
 	FIELD field_568 leftwardVelocity F
 	FIELD field_569 forwardVelocity F
 	FIELD field_570 isJumping Z
+	METHOD method_389 (FFF)Z
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
 	METHOD method_390 heal (I)V
 		ARG 1 amount
 	METHOD method_391 getDropItem ()I
 	METHOD method_392 updateMovements ()V
 	METHOD method_393 onDeath (Lnet/minecraft/class_140;)V
 		ARG 1 killer
 	METHOD method_394 livingTick ()V
+	METHOD method_395 getAmbientSound ()Ljava/lang/String;
```

Comment:
Your `getIdleSound` is better, pls commit this and the tick variable with your better name thanks